defaults/severities.yaml

```diff
@@ -99,9 +99,9 @@ CassandraStressEvent: CRITICAL
 GeminiStressEvent: CRITICAL
 ScyllaServerStatusEvent: NORMAL
 BootstrapEvent: NORMAL
-RepairEvent: NORMAL
+RepairEvent: DEBUG
```

Comment:
why are these being set as DEBUG, and not like either NORMAL?

Reply:
performance jobs need these in sct.log (it is used to decode the `Reactor stall` events

Reply:
@fgelcer , decoding events are picking up from db-node/messages.log, not from sct.log

Reply:
regular jobs decoding, yes, for performance jobs, it doesn't (yet) decode them automatically...
and right now sct.log is the file used to decode all the `Reactor stall` events, using `stall-analyser.py` tool inside seastar repo...